.github/workflows/publish_release.yml

```diff
@@ -0,0 +1,27 @@
+name: Publish Release
+env:
+  PYTHON_VERSION: "3.9"
```

Comment:
a bit aggressive, no? :)
we currently test in 3.6

Reply:
👍👍👍